riptide-failsafe/src/main/java/org/zalando/riptide/failsafe/CompositeTaskDecorator.java

```diff
@@ -0,0 +1,20 @@
+package org.zalando.riptide.failsafe;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import net.jodah.failsafe.function.ContextualSupplier;
+
+@AllArgsConstructor
+public class CompositeTaskDecorator implements TaskDecorator {
```

Comment:
```suggestion
private final class CompositeTaskDecorator implements TaskDecorator {
```

Reply:
+ a static factory method: `TaskDecorator.composite(TaskDecorator...)` and `TaskDecorator.composite(Iterable<TaskDecorator>)`

Reply:
Will do